### PR TITLE
Added bcl2fastq to travis

### PR DESCRIPTION
**Pull request by johandahlberg**
Moved the tests around, and made the changes necessary for this to merge into master after the arteria-core stuff was added.

### CODE REVIEW

File: bcl2fastq/bcl2fastq/lib/bcl2fastq_utils.py [hunk 1]
```diff
@@ -77,8 +77,11 @@ class BCL2FastqRunnerFactory:
     and the it's known binaries.
     """
 
-    def __init__(self, config_file = None):
-        self.bcl2fastq_mappings = Config.load_config(config_file)["bcl2fastq"]["versions"]
+    def __init__(self, config=None):
+        if config:
+            self.bcl2fastq_mappings = config["bcl2fastq"]["versions"]
+        else:
+            self.bcl2fastq_mappings = Config.load_config()["bcl2fastq"]["versions"]
```
Comment (withrocks) [inline bcl2fastq/bcl2fastq/lib/bcl2fastq_utils.py]: How about (for readability):

    def __init__(self, config=None):
        config = config or Config.load_config()
        self.bcl2fastq_mappings = config["bcl2fastq"]["versions"]

Comment (johandahlberg) [inline bcl2fastq/bcl2fastq/lib/bcl2fastq_utils.py]: Good point. Will fix that.

File: bcl2fastq/scripts/bcl2fastq-wsd [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+# Required-Stop:
+# Default-Start:     2 3 4 5
+# Default-Stop:
+# Short-Description: Provides a web service to start bcl2fastq
+# Description: Provides a web service interface to bcl2fastq
+### END INIT INFO
+
+
+# Execute in the /opt/runfolder virtualenv:
+source /opt/runfolder/bin/activate
```
Comment (withrocks) [inline bcl2fastq/scripts/bcl2fastq-wsd]: Note: This dependency on the runfolder service will be removed in a future checkin, when we move to Supervisor. This currently works, as long as the runfolder service is installed before this service.

Comment (johandahlberg) [inline bcl2fastq/scripts/bcl2fastq-wsd]: I'm not really using this at the moment, so I even considered removing it completely. But yeah - let's fix this later.

File: bcl2fastq/tests/test_utils.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+__author__ = 'johda411'
```
Comment (withrocks) [inline bcl2fastq/tests/test_utils.py]: Should this be there?

Comment (johandahlberg) [inline bcl2fastq/tests/test_utils.py]: No - it's something that pycharm keeps adding. I need change that setting.

**Comment by withrocks** [comment]
Looks good - is it ready to merge?

**Comment by johandahlberg** [comment]
I'll clean up the author tags and let you know once it's ready to merge.

**Comment by johandahlberg** [comment]
This is good to go now.

**Pull request merged**